css/style.css

```diff
@@ -1,3 +1,52 @@
-h1 {
+/*h1 {
   color: salmon;
+}*/
+*{
+	text-align: center;
+	/*border: 1px solid red;*/
+	box-sizing: border-box;
+	font-family: 'Roboto', sans-serif;
+}
+body{background: rgba(1,1,1,0.05);}
+main{
+	display: flex;
+	flex-direction: column;
+}
+.block >h2{
+	font-size: 180%;
+	margin-top: 70px;
+	color: rgb(120, 120, 120);
+}
+.block > p{
+	margin-bottom: -50px;
+	color: rgb(120, 120, 120);
+}
+img{
+	width: 100%;
+	height: 190px;
+}
+.imgDiv{
+	width: 30%;
+}
+h4{
+	color: white;
+	position: relative;
+	font-size: 140%;
+	letter-spacing: 1px;
+	top: 145px;
+}
+.imgBlock{
+	display: flex;
+	flex-direction: row;
```

Comment:
`flex-direction: row` is applied to all flex containers by default, so you don't have to write it out unless you're overriding an already existing `flex-direction` property.